canonical-web-and-design/jaas-dashboard
# Persist user menu across different pages

## Done

- Add UI redux key
- Persist user menu across different pages

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Open user menu
- Click 'Controllers' and verify that user menu remains open
- Click to the model details page and verify that user menu collapses with side navigation

## Details

Fixes #366
